test/jdk/javax/swing/JRadioButton/8033699/bug8033699.java

```diff
@@ -59,7 +59,9 @@ public class bug8033699 {
 
     public static void main(String[] args) throws Throwable {
         robot = new Robot();
-
+        SwingUtilities.invokeAndWait(() -> {
+            focusManager = KeyboardFocusManager.getCurrentKeyboardFocusManager();
+        });
```

Comment:
```suggestion
        SwingUtilities.invokeAndWait(() ->
                focusManager = KeyboardFocusManager.getCurrentKeyboardFocusManager());
```
Braces are redundant for one statement (in lambdas), or _“Statement lambda can be replaced with expression lambda.”_